lib/bundler/audit/scanner.rb

```diff
@@ -137,24 +205,80 @@ def scan_sources(options={})
       #
       # @since 0.4.0
       #
-      def scan_specs(options={})
+      def scan_specs(options={},&block)
+        scan_inner(@lockfile.specs,'gem',options,&block)
+      end
+
+      private
+
+      #
+      # Checks the Ruby version.
+      #
+      # @return [String]
+      #   The current Ruby version.
+      #
+      def ruby_version
+        # .gsub to separate strings (e.g., 2.1.0dev -> 2.1.0.dev,
+        # 2.2.0preview1 -> 2.2.0.preview.1).
+        `ruby -v`.split[1]
```

Comment:
Interestingly, that doesn't behave the way one might expect it to:
```
% ruby -v
ruby 2.2.3p173 (2015-08-18 revision 51636) [x86_64-darwin15]
% ruby --version
ruby 2.2.3p173 (2015-08-18 revision 51636) [x86_64-darwin15]
% ruby --verbose
%
```
I'll change both `-v` arguments to `--version` for clarity.